cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1573,4 +1573,11 @@
     </description>
   </property>
 
+  <property>
+    <name>metadata.max_chars.allowed</name>
```

Comment:
``metadata.max.allowed.chars``? Also, please talk to @awholegunch about incorporating this into his PR.